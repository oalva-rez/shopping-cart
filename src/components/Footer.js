import React from "react";

export default function Footer() {
  return (
    <footer>
      <div className="footer-info">
        <div>
          <h3>ABOUT THE SHOP</h3>
          <p>
            After being frustrated with being unable to find fitted high quality
            essentials at a fair price, we decided to take matters in to our own
            hands.
          </p>
        </div>
        <div>
          <h3>HELP</h3>
          <ul>
            <li>FAQ</li>
            <li>Returns</li>
            <li>Care Instructions</li>
            <li>Email Support</li>
            <li>Changing ot Modifying Order</li>
            <li>Terms & Conditions</li>
            <li>Privacy Policy</li>
            <li>Return Policy</li>
            <li>International Shipping Policy</li>
          </ul>
        </div>
        <div>
          <h3>NEWSLETTER</h3>
          <p>
            Subscribe to receive updates, access to exclusive deals, and more.
          </p>
          <input type="email" placeholder="Enter your email address" />
          <button>SUBSCRIBE</button>
        </div>
      </div>
      <p>Copyright &#169; Ozkar Alvarez</p>
    </footer>
  );
}
