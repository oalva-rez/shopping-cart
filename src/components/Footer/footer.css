.footer-rights {
  padding: 10px;
  text-align: center;
  background-color: black;
  color: rgb(211, 211, 211);
  font-size: 0.8rem;
  height: 100%;
}
.footer-info {
  display: flex;
  height: 100%;
  box-shadow: 0px 5px 5px #888, 0px -2px 4px rgba(0, 0, 0, 0.1);
}
.footer-info > * {
  flex: 1;
  margin: 0 10px;
}
.footer-about p {
  width: 70%;
}
.footer-help ul {
  padding: 0;
}
.footer-help li {
  list-style: none;
  margin-top: 4px;
  cursor: pointer;
}

.footer-newsletter {
  display: flex;
  flex-direction: column;

  margin-top: 20px;
}

.footer-newsletter p {
  width: 60%;
}
.footer-newsletter input {
  width: 40%;
  height: 40px;
  font-size: 1rem;
  padding: 3px 10px;
  border-radius: 0;
}
.footer-newsletter button {
  width: 30%;
  height: 40px;
  font-size: 1rem;
  padding: 3px 10px;
  border: none;
  outline: none;
  background-color: black;
  color: white;
  margin-top: 15px;
  cursor: pointer;
}
