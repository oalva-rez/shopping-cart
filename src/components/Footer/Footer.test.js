import { render, screen } from "@testing-library/react";
import Footer from "./Footer";
import "@testing-library/jest-dom";

describe("Footer", () => {
  it("matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
