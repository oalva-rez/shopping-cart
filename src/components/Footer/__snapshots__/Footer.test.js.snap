// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<div>
  <footer>
    <div
      class="footer-info"
    >
      <div
        class="footer-about"
      >
        <h3>
          ABOUT THE SHOP
        </h3>
        <p>
          After being frustrated with being unable to find fitted high quality essentials at a fair price, we decided to take matters in to our own hands.
        </p>
      </div>
      <div
        class="footer-help"
      >
        <h3>
          HELP
        </h3>
        <ul>
          <li>
            FAQ
          </li>
          <li>
            Returns
          </li>
          <li>
            Care Instructions
          </li>
          <li>
            Email Support
          </li>
          <li>
            Changing ot Modifying Order
          </li>
          <li>
            Terms & Conditions
          </li>
          <li>
            Privacy Policy
          </li>
          <li>
            Return Policy
          </li>
          <li>
            International Shipping Policy
          </li>
        </ul>
      </div>
      <div
        class="footer-newsletter"
      >
        <h3>
          NEWSLETTER
        </h3>
        <p>
          Subscribe to receive updates, access to exclusive deals, and more.
        </p>
        <input
          placeholder="Enter your email address"
          type="email"
        />
        <button>
          SUBSCRIBE
        </button>
      </div>
    </div>
    <p
      class="footer-rights"
    >
      Copyright Â© Ozkar Alvarez
    </p>
  </footer>
</div>
`;
